CHANGELOG.md

```diff
@@ -0,0 +1,50 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
Can CHANGELOG generation be automated in anyway? Or is it just simply about agreeing on rules and otherwise updating changelog manually?

Reply:
I think as of now it's manual since they can be different from commit messages (like something deprecated/upgrade instructions etc.)